### PR TITLE
Fixed CircleCI build that was failing due to newer images of PostgreSQL requiring a password.

### PR DESCRIPTION
**Pull request by wkozlik**
https://discuss.circleci.com/t/postgresql-image-password-not-specified-issue/34555

### CODE REVIEW

File: postgres/init.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ echo 'Setting up PostgreSQL database'
 # Replace with extracting username and password from config
 # Set password and utilize username from extracted var
 psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" <<-EOSQL
-    CREATE USER grafeas;
+    CREATE USER grafeas WITH PASSWORD 'changeme';
```
Comment (aysylu) [inline postgres/init.sh:9]: Does this need to be the same password as in `docker-compose.yaml`?

**Review by aysylu — CHANGES_REQUESTED** [review]

File: postgres/init.sh (hunk 1 above)
Comment (wkozlik) [reply postgres/init.sh:9]: The `grafeas` user with the password `changeme` is used in the server configuration https://github.com/grafeas/grafeas-pgsql/blob/master/server/config.yaml#L37. 
The `docker-compose.yaml` has a postgres user/pswd to start the postgres instance.
We could make it all the same if this leads to confusion.

Comment (aysylu) [reply postgres/init.sh:9]: Ah, gotcha, good point. Let's leave this as is for now.

**Review by aysylu — APPROVED** [review]

**Comment by aysylu** [comment]
Thanks for the fix!

**Pull request merged**